\section{Materials and Methods}

\subsection{Clade selection and phylogeny reconstruction}

Our criteria were that each clade (1) included a substantial number of species that occur in the Hengduan Mountains, as well as their closest known relatives in other biogeographic regions; (2) had sufficient molecular data available to infer a phylogeny that was broadly representative of the clade's taxonomic diversity and geographic range; and (3) had fossil data suitable for molecular clock calibration or secondary calibrations inferred from fossil dated phylogenies. We found 18 clades---16 of angiosperms and one each of gymnosperms (Pinaceae) and ferns (Microsorioideae)---that met these criteria. The combined taxon sample included a total of 3,262 ingroup species, of which 899 occur in the Hengduan Mountains region, 624 in the Himalayas-QTP, and 1,165 in the remainder of temperate/boreal East Asia (see \textit{Delimitation of biogeographic regions}). Of these, about 370 species are shared by the Hengduan Mountains and Himalayas-QTP regions. Across clades, the proportion of species sampled ranged from 20--97\% globally and 40--97\% for the Hengduan Mountains region (Table S1).

For each clade, we assembled molecular sequence alignments (Table S2; \textit{Appendix 1}) and fossil calibration data, and used relaxed molecular clock models implemented in BEAST 1.8 \citep{Drummond2012} or 2.0 \citep{Bouckaert2014} to generate a Bayesian posterior sample of time-calibrated phylogenies and the associated maximum-clade-credibility tree.  Detailed information is provided in \textit{SI Materials and Methods}.

%, representing approximately XX\%, YY\%, and ZZ\% of the vascular flora of each region (SI REF).

\subsection{Inference of range evolution and lineage diversification}

\subsubsection{Delimitation of biogeographic regions}

We defined 11 biogeographic regions (Fig.~S1), balancing considerations of model complexity, the need to accommodate the global distributions of the selected clades, and the granularity of our species range data. Of primary importance was distinguishing the Hengduan Mountains region from the geologically older parts of the QTP, especially the Himalayas. To that end, our definition of the Hengduan Mountains region follows \citet{Boufford2014}, and is bounded to the west by the Yarlung Tsangpo River in eastern Xizang (Tibet), to the northwest by the high plateau in Qinghai, to the north by the Tao River in southern Gansu, to the east by the Sichuan Basin, and to the south by subtropical forests and the Yunnanâ€“Guizhou Plateau. As a primary point of comparison, we defined the ``Himalayas-QTP'' region as including the plateau itself, the Himalayas to the south, the Kunlun Mountains to the north, and the Qilian Mountains to the northeast. The other regions we defined are: temperate/boreal East Asia (the eastern boreal part of Russia and temperate regions of East Asia, including Japan and Taiwan); central/western Asia (including the Xinjiang Uyghur Autonomous Region to the south of Kunlun Range); southeast Asia (including tropical regions of China, Malesia, and Papuasia); Australasia (Australia, New Zealand, and the southwestern Pacific); India (south of the Himalyas, including Sri Lanka); Africa; Europe; North America; and South America (south of the Panama Canal). We scored the geographic range of each sampled species as its presence or absence in these regions based on floras, online databases, and published data sets (\textit{Dataset S1}; see also \textit{SI Materials and Methods}).

\subsubsection{Ancestral range reconstruction}

We set up a DEC model of range evolution in Lagrange \citep{Ree2005,Ree2008}, specifying a region-adjacency matrix that defined the valid set of spatially contiguous ranges. We explicitly avoided placing any temporal constraints on dispersal, so that biogeographic inferences were independent of prior beliefs about the ages of the Hengduan Mountains or Himalayas-QTP regions. For each clade, we used the model to infer a distribution of phylogenetic histories of biogeographic events. A single history for a clade was generated by sampling a chronogram from its Bayesian posterior distribution, estimating the maximum-likelihood set of ancestral ranges (geographic speciation scenarios) at internal nodes, and randomly interpolating parsimonious sequences of dispersal and local extinction events along branches having different ancestral and descendant ranges. This yielded a complete chronology of biogeographic events, i.e., where and when ancestral lineages moved and underwent speciation and local extinction. To account for phylogenetic uncertainty, one history was generated for each of 500 chronograms drawn randomly from the posterior for each clade. For further details, see \textit{SI Materials and Methods}.

\subsubsection{Regional assembly processes through time}

We focused our analysis on the dynamics of the Hengduan Mountains region in comparison to the Himalayas-QTP and temperate East Asia. Our objective was to estimate, for each region, rates of \textit{in situ} diversification and colonization and their cumulative contributions to biotic assembly through time. Taking the biogeographic histories of clades from the ancestral-range analysis, we generated 500 pseudoreplicated joint histories---sets of one history drawn randomly from each clade's pool without replacement. From each joint history, we extracted the chronology of \textit{in situ} speciation, colonization, and local extinction events affecting the 3 regions of interest and binned them into 1 Myr periods. This allowed region-specific estimates of assembly processes through time. We calculated rolling estimates of \textit{in situ} speciation rates as $\lambda(t) = s(t)/n(t-1)$, where $s(t)$ is the number of \textit{in situ} speciation events inferred in a region in a 1-Myr period $t$ and $n(t-1)$ is the number of inferred lineages in the region in the previous period (the cumulative sum of \textit{in situ} speciation and colonization minus local extinction). We also calculated rolling rates of dispersal for each region as $d_{ij}(t) = c_{ij}(t)/n_i(t-1)$, where $c_{ij}(t)$ is the number of inferred colonization events of area $j$ from area $i$. For all estimates, confidence intervals (95\% quantiles) were calculated from the pseudoreplicated joint histories. Further details are provided in the \textit{SI Materials and Methods}.

\subsubsection{Geography-independent rates of diversification}

To complement the regional-process analysis and to better understand heterogeneity in rates of lineage diversification independently of geography, we generated Bayesian inferences of diversification rate for each clade using BAMM \cite{Rabosky2014}, which uses Markov chain Monte Carlo procedures to jointly estimate the number, parameters, and locations of distinct macroevolutionary regimes (rates of lineage birth and death, possibly time-dependent) on a phylogenetic tree. BAMM is able to account statistically for incomplete taxon sampling, so we assigned sampling fractions at the finest level of taxonomic resolution possible: this was generally at the level of genus, but at the level of section for \textit{Rhododendron} and \textit{Acer}. In cases where genera were not confidently resolved as monophyletic, we assigned sampling fractions at the higher taxonomic levels (tribe and subfamily). We ran each BAMM MCMC for 100 million generations with a sampling frequency of 1000 and assessed convergence by visually inspecting plots of the likelihood trace and calculating the effective sample size after discarding the first 10\% of the run as burn-in. We identified the 95\% credible set of distinct shift configurations and the overall best set of rate shifts given the data using the BAMMtools package in R \cite{Rabosky2014}.