%\section*{Methods}

\subsection*{Clade selection and phylogeny reconstruction}

Our criteria were that each clade (1) included a substantial number of
species that occur in the Hengduan Mountains, as well as their closest
known relatives in other biogeographic regions; (2) had sufficient
molecular data available to infer a phylogeny that was broadly
representative of the clade's taxonomic diversity and geographic
range; and (3) had fossil data suitable for molecular clock
calibration or secondary calibrations inferred from fossil dated
phylogenies. We found 19 clades---17 of angiosperms and one each of
gymnosperms (Pinaceae) and ferns (Microsorioideae)---that met these
criteria. The combined taxon sample included 4,668 ingroup species, of
which 930 occur in the Hengduan Mountains region, 703 in the
Himalayas-QTP, and 1,045 in the remainder of temperate/boreal East
Asia (see \textit{Delimitation of biogeographic regions}). Of these,
about 370 species are shared between the Hengduan Mountains and
Himalayas-QTP regions. Across clades, the proportion of species
sampled ranged from 20--97\% globally and 26--94\% for the Hengduan
Mountains region (Table S1).

For each clade, we assembled molecular sequence alignments
(\textit{Dataset S1}) and fossil calibration data, and used relaxed
molecular clock models implemented in BEAST
\citep{Drummond2012,Bouckaert2014} to generate a Bayesian posterior
sample of time-calibrated phylogenies and the associated
maximum-clade-credibility tree (MCC tree).  More detailed information
is provided in \textit{SI Text}.

%, representing approximately XX\%, YY\%, and ZZ\% of the vascular flora of each region (SI REF).

\subsection*{Inference of range evolution and lineage diversification}

\subsubsection*{Delimitation of biogeographic regions}

We defined 11 biogeographic regions (Fig.~S1) to accommodate the
global distributions of the selected clades, the granularity of
available species range data, and our primary focus on distinguishing
the Hengduan Mountains region from adjacent, geologically older parts
of the QTP, especially the Himalayas. To that end we defined the
Hengduan Mountains region following Boufford \citep{Boufford2014},
bounded to the west by the Yarlung Tsangpo River in eastern Xizang
(Tibet), to the northwest by the high plateau in Qinghai, to the north
by the Tao River in southern Gansu, to the east by the Sichuan Basin,
and to the south by subtropical forests and the Yunnanâ€“Guizhou Plateau
(Fig.~\ref{fig:map}).

We defined the complementary ``Himalayas-QTP'' region as including the
plateau itself, the Himalayas to the south, the Kunlun Mountains to
the north, and the Qilian Mountains to the northeast. Our decision to
<<<<<<< Updated upstream
lump the Himalayas and QTP into a single biogeographic unit, when it
is clear they have distinct geological histories, is not ideal but
reflects the fact that the granularity of available data on species
ranges makes it difficult to differentiate between the Himalayas and
the QTP; moreover, an analysis in which we attempt to do so yields
nearly identical results (see \textit{SI text}).
=======
treat the Himalayas and QTP into a single unit for biogeographic
analysis, when it is clear they have distinct geological histories, is
not ideal but reflects the fact that available data on species ranges
often lacked sufficient detail to differentiate between the Himalayas
and the QTP.
>>>>>>> Stashed changes

The other regions we defined are: temperate/boreal East Asia (the
eastern boreal part of Russia and temperate regions of East Asia,
including Japan and Taiwan); central/western Asia (including the
Xinjiang Uyghur Autonomous Region to the north of the Kunlun Range);
southeast Asia (including tropical regions of China, Malesia, and
Papuasia); Australasia (Australia, New Zealand, and the southwestern
Pacific); India (south of the Himalyas, including Sri Lanka); Africa;
Europe; North America; and South America (south of the Panama
Canal). We scored the geographic range of each sampled species as its
presence or absence in these regions based on floras, online
databases, and published data sets (see \textit{SI Text} and
\textit{Dataset S2}).

\subsubsection*{Ancestral range reconstruction}

We estimated ancestral geographic ranges for all ingroups using a
dispersal-extinction-cladogenesis (DEC) model in Lagrange
\citep{Ree2005,Ree2008} in which dispersal was allowed based on
spatial adjacency (Fig.~S1), and the maximum range size was set to
3. We explicitly avoided placing any temporal constraints on
dispersal, so that biogeographic inferences were independent of prior
beliefs about the ages of the Hengduan Mountains or Himalayas-QTP
regions. For each ingroup, we inferred a distribution of biogeographic
histories to account for phylogenetic uncertainty. A single history
was generated in three steps. First, a chronogram was randomly sampled
from the clade's Bayesian posterior distribution. Second, the
maximum-likelihood set of ancestral ranges (geographic speciation
scenarios) at internal nodes was estimated using Lagrange. Third,
parsimonious sequences of dispersal and local extinction events were
randomly interpolated along branches having different ancestral and
descendant ranges. This procedure yielded a complete chronology of
biogeographic events, i.e., where and when ancestral lineages moved
and underwent speciation and local extinction. To account for
phylogenetic uncertainty, one history was generated for each of 500
chronograms sampled from the posterior for each clade.

\subsubsection*{Regional assembly processes through time}

We focused our analysis on the dynamics of the Hengduan Mountains
region in comparison to the Himalayas-QTP and temperate East Asia. Our
objective was to estimate, for each region, rates of \textit{in situ}
diversification and colonization and their cumulative contributions to
biotic assembly through time, while accounting for phylogenetic
uncertainy. Taking the biogeographic histories of clades from the
ancestral-range analysis, we generated 500 pseudoreplicated joint
histories---sets of one history drawn randomly from each clade's pool
without replacement. From each joint history, we extracted the
chronology of \textit{in situ} speciation, colonization, and local
extinction events affecting the Hengduan Mountains, Himalayas-QTP, and
temperate/boreal East Asia, and binned them into 1 Myr periods. This
allowed region-specific estimates of assembly processes through
time. We calculated rolling estimates of per capita \textit{in situ}
speciation rates as $\lambda(t) = s(t)/n(t-1)$, where $s(t)$ is the
number of \textit{in situ} speciation events inferred in a region in a
1-Myr period $t$ and $n(t-1)$ is the number of inferred lineages in
the region in the previous period (the cumulative sum of \textit{in
  situ} speciation and colonization minus local extinction). We also
calculated rolling per capita rates of colonization between regions as
$d_{ij}(t) = c_{ij}(t)/n_i(t-1)$, where $c_{ij}(t)$ is the number of
inferred colonization events of area $j$ from area $i$. For all
estimates, confidence intervals (5--95\% quantiles) were calculated
from the pseudoreplicated joint histories. Our results thus do not
condition on any particular tree topologies or branch lengths, and
reflect the levels of clade support, and confidence in divergence
times, provided by the sequence data.

\subsubsection*{Geography-independent rates of diversification}

To complement the regional-process analysis and to better understand
heterogeneity in rates of lineage diversification independently of
geography, we generated Bayesian inferences of diversification rate
for each clade's maximum-clade-credibility tree, with posterior mean
branch lengths, using BAMM version 2.5 and the BAMMtools R package
\citep{Rabosky2014}. BAMM uses Markov chain Monte Carlo procedures to
jointly estimate the number, parameters, and locations of distinct
macroevolutionary regimes (rates of lineage birth and death, possibly
time-dependent) on a phylogenetic tree. It can account statistically
for incomplete taxon sampling, so we assigned sampling fractions at
the finest level of taxonomic resolution possible (see \textit{SI
  text}). Priors for speciation and extinction were set empirically
using the \textrm{setBAMMpriors} function. For all clades smaller than
500 species, we set the geometric prior on the expected number of
regime shifts to 1, as recommended in the BAMM documentation; for
larger clades we also tested a prior of 10 expected shifts. We ran
each BAMM MCMC analysis for 10 million generations with a sampling
frequency of 1/1000 and assessed convergence by visually inspecting
plots of the likelihood trace and calculating the effective sample
size after discarding the first 10\% of the run as burn-in. For each
clade, we calculated Bayes factors for the distinct numbers of regime
shifts sampled, marginal odds ratios (relative evidence in favor of a
shift) for individual branches, and the 95\% credible set of distinct
shift configurations.

%%% Local Variables:
%%% mode: latex
%%% TeX-master: "master"
%%% End:
