\section{Materials and Methods}

\subsection{Clade selection and phylogeny reconstruction}

We selected 18 clades of vascular plants that best met our criteria: each clade (1) included a substantial number of species that occur in the Hengduan Mountains, as well as species that occur elsewhere; (2) had sufficient molecular data available to infer a phylogeny that is broadly representative of the clade's taxonomic diversity and geographic range; and (3) had fossil data suitable for molecular clock calibration. The clades include angiosperms, gymnosperms, and ferns, and together include XXX species, of which YYY could be sampled for phylogeny reconstruction (SI REF). For each clade, we assembled molecular sequence alignments and fossil data and used BEAST (REF) to generate a Bayesian posterior sample of time-calibrated phylogenies and the associated maximum-clade-credibility tree (SI REF).

\subsection{Ancestral range histories}

We defined 11 biogeographic regions to accommodate the global distribution of our clade set and scored the geographic range of each sampled species as its presence or absence in these regions. We constructed a DEC model of range evolution in Lagrange (REF), specifying a region-adjacency matrix that defined the valid set of spatially contiguous ranges, with a maximum range size of 5. For each clade, we used the model to infer a distribution of phylogenetic histories of range evolution. A single history was generated by sampling a chronogram from the Bayesian posterior distribution, estimating its maximum-likelihood set of ancestral ranges at internal nodes, and randomly interpolating parsimonious sequences of dispersal and local extinction events along branches having different ancestral and descendant ranges; this yielded a complete chronology of anagenetic and cladogenetic range evolution events. To account for phylogenetic uncertainty, a single history was generated for each of 500 chronograms drawn randomly from the Bayesian posterior distribution for each clade. For further details, see SI.

\subsection{Regional assembly processes through time}

Our objective was to estimate, for each region of interest $i$, the rate of \textit{in situ} diversification through time $r_i(t)$ and the rate of colonization through time, $m_{ij}$