\section{Materials and Methods}

\subsection{Clade selection and phylogeny reconstruction}

We selected 18 clades of vascular plants that best met our criteria: each clade (1) included a substantial number of species that occur in the Hengduan Mountains, as well as species that occur elsewhere; (2) had sufficient molecular data available to infer a phylogeny that is broadly representative of the clade's taxonomic diversity and geographic range; and (3) had fossil data suitable for molecular clock calibration. The clades include angiosperms, gymnosperms, and ferns, and together include XXX species, of which YYY could be sampled for phylogeny reconstruction (SI REF). For each clade, we assembled molecular sequence alignments and fossil data and used BEAST (REF) to generate a Bayesian posterior sample of time-calibrated phylogenies and the associated maximum-clade-credibility tree (SI REF).

\subsection{Biogeographic histories of clades}

We defined 11 biogeographic regions to accommodate the global distribution of our clade set and scored the geographic range of each sampled species as its presence or absence in these regions. We constructed a DEC model of range evolution in Lagrange (REF), specifying a region-adjacency matrix that defined the valid set of spatially contiguous ranges, with a maximum range size of 5. For each clade, we used the model to infer a distribution of phylogenetic histories of biogeographic events. A single history for a clade was generated by sampling a chronogram from its Bayesian posterior distribution, estimating the maximum-likelihood set of ancestral ranges (geographic speciation scenarios) at internal nodes, and randomly interpolating parsimonious sequences of dispersal and local extinction events along branches having different ancestral and descendant ranges; this yielded a complete chronology of biogeographic events, i.e., where and when ancestral lineages moved, speciated, and went locally extinct. To account for phylogenetic uncertainty, one history was generated for each of 500 chronograms drawn randomly from the posterior for each clade. For further details, see SI.

\subsection{Regional assembly processes through time}

We focused our analysis on the dynamics of the Hengduan Mountains in comparison to the Himalayas and temperate East Asia. Our objective was to estimate, for each region, rates of \textit{in situ} diversification and colonization and their cumulative contributions to biotic assembly through time. Taking the biogeographic histories of clades from the ancestral-range analysis, we generated 100 pseudoreplicated joint histories by drawing one history from each clade's pool, without replacement, for each pseudoreplicate. From each joint history, we extracted the events affecting the 3 regions of interest and binned them into 1 Myr periods from 80 Mya to the present. This allowed us to calculate, for each pseudoreplicate, rolling estimates of \textit{in situ} speciation and colonization in events per million years for each region. Further details are provided in the SI.