\section{Materials and Methods}

\subsection{Clade selection and phylogeny reconstruction}

We selected 18 clades of vascular plants that best met our criteria: each clade (1) included a substantial number of species that occur in the Hengduan Mountains, as well as species that occur elsewhere; (2) had sufficient molecular data available to infer a phylogeny that is broadly representative of the clade's taxonomic diversity and geographic range; and (3) had fossil data suitable for molecular clock calibration. The clades include angiosperms, gymnosperms, and ferns, and together include XXX species, of which YYY could be sampled for phylogeny reconstruction (SI REF). For each clade, we assembled molecular sequence alignments and fossil data and used BEAST (REF) to generate a Bayesian posterior sample of time-calibrated phylogenies and the associated maximum-clade-credibility tree (SI REF).

\subsection{Ancestral range inference}

We defined 11 biogeographic regions to accommodate the global distribution of our clade set and scored the geographic range of each sampled species as its presence or absence in these regions. For each clade, we inferred phylogenetic histories of range evolution using the DEC model implemented in Lagrange (Ree and Smith 2008). To accommodate phylogenetic uncertainty, we drew 500 chronograms randomly from the Bayesian posterior sample; on each we generated a complete chronology of range evolution events by augmenting the maximum-likelihood set of ancestral range inheritance scenarios at internal nodes with random parsimonious interpolations of dispersal and local extinction events along the branches of the tree. For details, see SI.

\subsection{Regional assembly processes through time}

Our objective was to estimate, for each region of interest $i$, the rate of \textit{in situ} diversification through time $r_i(t)$ and the rate of colonization through time, $m_{ij}$