\section{Materials and Methods}

\subsection{Clade selection and phylogeny reconstruction}

Our criteria were that each clade (1) included a substantial number of species that occur in the Hengduan Mountains, as well as their closest known relatives in other biogeographic regions; (2) had sufficient molecular data available to infer a phylogeny that was broadly representative of the clade's taxonomic diversity and geographic range; and (3) had fossil data suitable for molecular clock calibration. For each clade, we assembled molecular sequence alignments and fossil calibration data and used a relaxed molecular clock model implemented in BEAST (REF) to generate a Bayesian posterior sample of time-calibrated phylogenies and the associated maximum-clade-credibility tree (SI REF).

We found 16 clades of angiosperms and one each of gymnosperms (Pinaceae) and ferns (Microsorioideae) that met our criteria. The combined taxon sample included a total of 3,266 ingroup species, of which 899 occur in the Hengduan Mountains, 628 in the Himalayas, and 1,165 in temperate East Asia. XXX species occur in both the Hengduan Mountains and Himalayas. Across clades, the proportion of species sampled ranged from XX--YY\% (REF:SI Table).%, representing approximately XX\%, YY\%, and ZZ\% of the vascular flora of each region (SI REF).

\subsection{Inference of biogeographic histories}

We defined 11 biogeographic regions to accommodate the global distribution of our clade set and scored the geographic range of each sampled species as its presence or absence in these regions. We constructed a DEC model of range evolution in Lagrange (REF), specifying a region-adjacency matrix that defined the valid set of spatially contiguous ranges, with a maximum range size of 5. We explicitly avoided placing any temporal constraints on dispersal, so that biogeographic inferences were independent of prior beliefs about the ages of the Hengduan Mountains and Himalayas. For each clade, we used the model to infer a distribution of phylogenetic histories of biogeographic events. A single history for a clade was generated by sampling a chronogram from its Bayesian posterior distribution, estimating the maximum-likelihood set of ancestral ranges (geographic speciation scenarios) at internal nodes, and randomly interpolating parsimonious sequences of dispersal and local extinction events along branches having different ancestral and descendant ranges. This yielded a complete chronology of biogeographic events, i.e., where and when ancestral lineages moved, speciated, and went locally extinct. To account for phylogenetic uncertainty, one history was generated for each of 500 chronograms drawn randomly from the posterior for each clade. For further details, see SI.

\subsection{Regional assembly processes through time}

We focused our analysis on the dynamics of the Hengduan Mountains in comparison to the Himalayas and temperate East Asia. Our objective was to estimate, for each region, rates of \textit{in situ} diversification and colonization and their cumulative contributions to biotic assembly through time. Taking the biogeographic histories of clades from the ancestral-range analysis, we generated 500 pseudoreplicated joint histories---sets of one history drawn randomly from each clade's pool without replacement. From each joint history, we extracted the chronology of \textit{in situ} speciation, colonization, and local extinction events affecting the 3 regions of interest and binned them into 1 Myr periods. This allowed region-specific estimates of assembly processes through time. We calculated rolling estimates of \textit{in situ} speciation rates as $\lambda(t) = s(t)/n(t-1)$, where $s(t)$ is the number of \textit{in situ} speciation events inferred in a region in a 1-Myr period $t$ and $n(t-1)$ is the number of inferred lineages in the region in the previous period (the cumulative sum of \textit{in situ} speciation and colonization minus local extinction). We also calculated rolling rates of dispersal for each region as $d_{ij}(t) = c_{ij}(t)/n_i(t-1)$, where $c_{ij}(t)$ is the number of inferred colonization events of area $j$ from area $i$. For all estimates, confidence intervals (quantiles) were calculated from the pseudoreplicated joint histories. Further details are provided in the SI.